src/Libraries/CoreNodes/DateTime.cs

```diff
@@ -50,6 +50,20 @@ public static System.DateTime Today
         }
 
         /// <summary>
+        ///     Return a specified date and time as a string, in the specified format.
+        /// </summary>
+        /// <param name="dateTime">The DateTime to format.</param>
+        /// <param name="format">String representation of the date format. Defaults to standard format "F", which outputs acording to operating system locale : "Monday, June 15, 2009 1:45:30 PM" for (en-US).
+        /// Search "MSDN Custom Date and Time Format Strings" for a comprehensive list of format specifiers.
+        ///</param>
+        /// <returns name="string">The specified date and time as a string, in the specified format.</returns>
+        public static string Format(System.DateTime dateTime, string format="F")
+        {
+            if (System.String.IsNullOrEmpty(format)) format = "F";
+            return dateTime.ToString(format, CultureInfo.InvariantCulture);
```

Comment:
Looks like using of CultureInfo.InvariantCulture makes the format independent with the operating system locale, do you mind me modifying the tooltip for the Format above?